tests/BenchmarkDotNet.IntegrationTests/ValuesReturnedByBenchmarkTest.cs

```diff
@@ -30,6 +30,12 @@ public Diagnostics.Windows.InliningDiagnoser TypeFromCustomDependency()
             {
                 return new Diagnostics.Windows.InliningDiagnoser();
             }
+
+            [Benchmark]
+            public Diagnostics.Windows.TailCallDiagnoser TypeFromCustomDeoendency1()
```

Comment:
The test which returns `InliningDiagnoser` verifies that we can have a benchmark which returns type from custom dll. Your test checks the same thing. It does not add any value and it prolongs the build. As of today we are very close of reaching the max time per build now so we try not to duplicate the tests. Please remove it